irohad/ametsuchi/impl/postgres_command_executor.cpp

```diff
@@ -190,21 +198,26 @@ namespace {
    * @param sql - connection on which to execute statement
    * @param cmd - sql query to be executed
    * @param command_name - which command executes a query
+   * @param query_args - lambda to get a string representation of query
```

Comment:
since query_args is an std::function, it is not correct to call it a lambda, since it can be any callable object. Please change the comment on this and on all functions which take it as an argument